### PR TITLE
inverted T axis, added minor ticks

### PR DESCRIPTION
**Pull request by petiay**
Also, removed the transpose of the T vs L plot in the HR diagram. Not sure why the transpose was being plotted since this was reversing the T and L axes yet the axis labels were not reversed (that is, the x-axis was still T and the y-axis was still L).

### CODE REVIEW

**Comment by karllark** [comment]
Can you provide the before/after versions of this plot?  Useful to understand the change.

File: beast/plotting/plot_param_err.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from astropy.io import fits
 from matplotlib.colors import LogNorm
 from scipy.stats import binned_statistic_2d as stat2d
+from matplotlib.ticker import (MultipleLocator)
```
Comment (karllark) [inline beast/plotting/plot_param_err.py:6]: Very minor.  The () are not needed, especially with one import.

**Review by karllark — CHANGES_REQUESTED** [review]

File: beast/plotting/plot_param_err.py (hunk 1 above)
Comment (petiay) [reply beast/plotting/plot_param_err.py:6]: Yes! This is leftover from having more ticker imports, and having to remove them to correct code style errors.

**Comment by petiay** [comment]
> Can you provide the before/after versions of this plot? Useful to understand the change.

Here is a before/after. It turns out that plotting the transpose of log(T) vs log(L) with matplotlib's imshow() is needed after all, therefore it has been returned in an updated commit:
Plot 1: Made with the original code
Plot 2: A simple flip of the x-axis to plot an HR diagram

One addition I may make to this code would be to:
A. Either zoom into the HR as in the Rv panel of Plot 2 above OR
B. Choose a plotting scheme which shows all sources since the current scheme does not.
A) would remove some interesting sources, so perhaps it's not the best option.  Feedback is welcome.

Plot 1
param_unc_orig_transposed_Tincr

Plot 2
param_unc_transposed_Tdecr

**Comment by karllark** [comment]
So, plot 2 is what we get with this PR, right?

**Comment by petiay** [comment]
Correct!

**Review by karllark — APPROVED** [review]
Looks great.  Thanks!

**Pull request merged**